apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/Button/Button.stories.tsx

```diff
@@ -9,7 +10,13 @@ import { Button } from '.'
 const ButtonStory: Meta<typeof Button> = {
   ...journeysAdminConfig,
```

Comment:
We use `journeyAdminConfig` when we need snapshots of both mobile and desktop screen sizes. Since these button styles don't change based of that, we should change this to use `simpleComponentConfig`
```suggestion
  ...simpleComponentConfig,
```